lib/systems/src/PedalsSystem.cpp

```diff
@@ -1,82 +1,87 @@
-#include "PedalsSystem.h"
+#include <PedalsSystem.h>
```

Comment:
the standard is that angle brackets are for system libraries and quotes for programmer defined libraries:

https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename